src/index.ts

```diff
@@ -23,6 +27,7 @@ export const PetraWalletName = "Petra" as WalletName<"Petra">;
 
 export class PetraWallet implements AdapterPlugin {
   readonly name = PetraWalletName;
+  readonly version = "v2";
```

Comment:
having it here will make the adapter to use this plugin as `v2` for everything - types/logic/etc. just making sure we are ok with it